kernel/src/hil/wifi_cyw43.rs

```diff
@@ -0,0 +1,165 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright OxidOS Automotive 2025.
+
+//! Interfaces for CYW43439 WiFi devices
```

Comment:
Can we document here how these traits relate to each other and how they should be used?